view-user/src/components/common/Modal/Modal.tsx

```diff
@@ -0,0 +1,36 @@
+import { type ReactNode } from "react";
+import styles from "./Modal.module.css";
+
+interface ModalProps {
+  children: ReactNode;
+  isOpened: boolean;
+  close: () => void;
+  canCloseByClickingBackground?: boolean;
+}
+
+const Modal = ({
+  children,
+  isOpened,
+  close,
+  canCloseByClickingBackground = true,
+}: ModalProps) => {
+  return (
+    <>
+      {isOpened && (
+        <div className={styles.wrapper}>
+          <div className={styles.content}>
+            <button className={styles.btnClose} onClick={close}>
+              ✕
```

Comment:
```suggestion
<RxCrossCircled />
```
react-icons使いましょう
https://react-icons.github.io/react-icons/search?q=cross